lib/board.rb

```diff
@@ -1,4 +1,5 @@
-class Board :cells
+class Board
+  attr_reader :cells
```

Comment:
I didn't create an attr_reader properly and I couldn't figure out why it looked weird!